nuxeo-features/nuxeo-automation/nuxeo-automation-core/src/test/java/org/nuxeo/ecm/automation/core/test/CoreOperationsTest.java

```diff
@@ -638,6 +653,26 @@ public void testCreateVersion3() throws Exception {
         }
     }
 
+    @Test
+    public void testCheckinVersion() throws Exception {
+        List<String> expected = List.of("none;0.0", "minor;0.1", "major;1.0",
```

Comment:
I probably would find it easier to read with:
```java
public void testCheckinVersion() throws Exception {
    testCheckinVersion("none", "0.0");
    testCheckinVersion("minor", "0.1");
    ...
}
```
with the needed `testCheckinVersion(version, expected)` method, to avoid unnecessary `split` and `for`.